velox/functions/prestosql/tests/UuidTest.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+
+class UuidTest : public functions::test::FunctionBaseTest {};
+
+TEST_F(UuidTest, uuid) {
```

Comment:
Let's add some tests to ensure the signature of the new function is correct. See https://github.com/facebookincubator/velox/pull/2965 and TEST_F(DateTimeFunctionsTest, dateTruncSignatures) for an example.

Reply:
Done